projects/http-auth/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+	"html"
+	"io"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/joho/godotenv"
+	"golang.org/x/time/rate"
+)
+
+func main() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal("Error loading .env file")
```

Comment:
i would suggest including the actual error as well. Something like this should work:
log.Fatalf("Error loading .env file - %v", err)